verification/utils/slices/slices.gobra

```diff
@@ -141,6 +141,39 @@ func NilAcc_Bytes() {
 	fold Bytes(nil, 0, 0)
 }
 
+ghost
+requires len(s1) > 0 || len(s2) > 0
+preserves Bytes(s1, 0, len(s1))
+preserves acc(Bytes(s2, 0, len(s2)), _)
+ensures s1 !== s2
+decreases
+func PermsImplyIneqWithWildcard(s1 []byte, s2 []byte) {
+	unfold Bytes(s1, 0, len(s1))
+	unfold acc(Bytes(s2, 0, len(s2)), _)
+	if len(s1) > 0 && len(s2) > 0 {
+		assert &s1[0] != &s2[0]
```

Comment:
Can you add a comment explaining that you have this here to trigger the qtfier bodies?